Pizza Divider.js

```diff
@@ -0,0 +1,14 @@
+const submitButton = document.getElementById('submit');
+submitButton.addEventListener('click', function(event) {
+let peopleNumber = document.getElementById('people').value;
+let pizzaNumber = document.getElementById('pizzas').value;
+let sliceNumber = document.getElementById('slices').value;
+let pizzaSlices = sliceNumber * pizzaNumber;
+let fullTotal = pizzaSlices / peopleNumber;
+let sharedSlices = Math.floor(fullTotal);
+let reminderSlices = fullTotal - sharedSlices;
+Math.round(reminderSlices *= peopleNumber);
```

Comment:
you are calculating a number here, but you forgot to assign the result to a variable :slightly_smiling_face:

Reply:
trying to be clean.......fail
Also, the reactions I left on this comment was a mistake

Reply:
written different code so it actual works properly now.